.github/workflows/ci-test-suites.yml

```diff
@@ -65,6 +64,7 @@ jobs:
     # TODO i#4549: Remove/replace this workaround when necessary support is added.
```

Comment:
Maybe add a note that this is present in every job below so someone doesn't just remove this one.

Reply:
Done.